libclamav_rust/.gitignore

```diff
@@ -0,0 +1 @@
+clamav_rust.h
```

Comment:
Can we have it build the binding in the `build/libclamav_rust` directory instead, so the build system doesn't contaminate the source directory?